java/BUCK

```diff
@@ -8,8 +8,8 @@ java_binary(
 java_library(
   name = 'watchman-lib',
   srcs = glob(['src/**/*.java']),
-  source = '1.6',
-  target = '1.6',
+  source = '7',
```

Comment:
Sorry to add more changes to this, but in generating an IJ project using `buck project`, I realized that the source level had been incorrectly set to 6, whereas the code is this library is actually written to 7.